WcaOnRails/app/views/static_pages/about.html.erb

```diff
@@ -15,14 +15,15 @@
   <p><%= t('about.structure.board_html') %></p>
 
   <ul>
-    <% @board_members.order(:name).each do |board_member| %>
-      <li><%= board_member.name %> - <%= board_member.region %></li>
+    <% Team.find_by_friendly_id(Team::BOARD_FRIENDLY_ID).current_members.includes(:user).map(&:user).sort_by(&:name).each do |board_member| %>
+      <li><%= board_member.name %> - <%= board_member.country.name %></li>
```

Comment:
The region field is for delegates. Board members are not necessarily delegates, so that field may be empty for a future member.